operator/tools/controller-gen/go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/stackrox/rox/operator/tools/controller-gen
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.9
+toolchain go1.23.4
 
-require sigs.k8s.io/controller-tools v0.16.5
+require sigs.k8s.io/controller-tools v0.17.0
```

Comment:
@porridge Do we need this file? We already have this dependency in `main`. Do we need two different versions?

https://github.com/stackrox/stackrox/blob/4b6c29c01904a54f475dcbb0278eda30ed016630/go.mod#L169

Reply:
- https://github.com/stackrox/stackrox/pull/13139

Reply:
- https://github.com/stackrox/stackrox/pull/13822

Reply:
Let me respond on the other PR.